### ISSUE TITLE
Back Button Issue

### ISSUE THREAD

**Issue by hn1** [metafizzy/infinite-scroll#950]
When using the back button, Infinite|Scroll goes to a single page. i.e. showing only partial results from what the use left the page at.

Is there any way to maintaine page position AND also have everything that the user had scrolled to and viewed also appear as well?

**Comment by roshangeorge97** [comment]
yes, mostly it can be fixed by toggling position between absolute and relative.

**Comment by hn1** [comment]
The demos show what I described as the problem and not a real scenario application.
Are there any examples. anywhere.
Thanks

**Comment by hn1** [comment]
By the way, if you kown anyone who knows how to do this properly, we are prepared to pay for their time to get this done for us.
Thanks

**Comment by malagebidi** [comment]
I need this too

**Comment by hn1** [comment]
Unfortunately not. There also does not seem to be any support of any kind from the developers of this product even for a paid version.
